test/test_clients.py

```diff
@@ -9,7 +9,7 @@
 from wampy.peers.clients import Client
 from wampy.roles.callee import callee
 from wampy.testing.helpers import wait_for_session
-
+from time import sleep
```

Comment:
small thing, but i prefer to abide by most pep8 style conventions. can you move this standard lib import above the wampy ones please? the line between them is good.